MOTIVATION.md

```diff
@@ -1,21 +1,21 @@
+The Java developer world has packages (jars, wars, ears...), tools ([Ant](https://ant.apache.org), [Maven](https://maven.apache.org), [Ivy](https://ant.apache.org/ivy), [Gradle](https://www.gradle.org/), ...) and established workflows to handle software distribution while Linux distros have their own ([zypper](https://en.opensuse.org/Portal:Zypper), [yum](https://en.wikipedia.org/wiki/Yum_(software)), [APT](https://en.wikipedia.org/wiki/APT_(software)), ...). Since the two communities have different goals and requirements, Linux distros typically want to repackage Java software with their own format, tools and workflows. Reasons range from ease of installation, predictable rebuildability, support to (security) patching, management tools, legal issues, etc.
 
 Unfortunately those two "schemes" became very different over time, so automatic translation/repackaging is not possible except in very simple cases. This leads to a lot of tedious, error-prone manual work that Linux packagers have to do in order to fit the "alien" Java model into distro packaging rules that were thought and optimized with a different ecosystem in mind.
 
 A typical example is packaging any software built by Maven on SUSE distros. A number of pain points arise:
 
-* Maven requires Internet access and downloads precompiled code as part of the build. RPMs have to be built on a standalone machine in [OBS](http://en.opensuse.org/openSUSE:Build_Service), and they should build code from sources they are provided beforehands exclusively;
-* [Maven is basically a plugin container](http://maven.apache.org/plugins/), so hundreds of different plugins have to be installed to build real-life projects (the exact plugin set and their dependencies is determined at build time). While this is no big deal for Java developers, since they get the corresponding jars prebuilt from public Intenet Maven repositories, it is a nightmare for distros, because all shipping code is supposed to be built from scratch and packaged!
+* Maven requires internet access and downloads precompiled code as part of the build. RPMs have to be built on a standalone machine in [OBS](https://en.opensuse.org/openSUSE:Build_Service), and they should build code from sources they are provided beforehand exclusively;
```

Comment:
TIL Internet is not supposed to be capitalized any longer - at least according to modern style guides.

https://en.wikipedia.org/wiki/Capitalization_of_Internet

Duh, I'm an old man!

 👴